docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -125,7 +125,20 @@ See the [Release Notes](https://www.intel.com/content/www/us/en/developer/articl
       cd /opt/intel/openvino_2022.3.1
       sudo -E ./install_dependencies/install_openvino_dependencies.sh
 
-6. For simplicity, it is useful to create a symbolic link as below:
+6. (Optional) Install *numpy* Python Library:
+
+   .. note::
+
+      This step is required only when you decide to use Python API.
+
+   You can use the ``requirements.txt`` file from the ``opt/intel/openvino_2022.3.1/python/python.<x>`` folder:
```

Comment:
Looks like we forgot leading slash here (and in other places too).
And we have to port it into 2023.0